provider/gce/environ_broker.go

```diff
@@ -131,6 +131,27 @@ func (env *environ) findInstanceSpec(
 	return spec, errors.Trace(err)
 }
 
+func (env *environ) imageURLBase(os jujuos.OSType) (string, error) {
+	base, useCustomPath := env.ecfg.baseImagePath()
+	if useCustomPath {
+		return base, nil
```

Comment:
Am not worried about testing for the OS here, as it has already been tested within the `getMetadata()` call within `getDisks()` anyway